Jenkinsfile

```diff
@@ -1 +1 @@
-buildPlugin(jenkinsVersions: [null, '2.60.3', '2.89'])
+buildPlugin(jenkinsVersions: [null, '2.138.1'])
```

Comment:
Current baseline is 2.60.3, so there's no point to test against it twice, and I figured 2.89 was probably added to test against newer Jenkins versions so I figured it made sense to update to the latest LTS.

Reply:
🤷‍♂️